mod_reforged/rf_sling_pot_skill.nut

```diff
@@ -0,0 +1,98 @@
+// This class is combines common properties of the new sling-an-item skills
+::Reforged.applySlingPotSkill <- function( o )
```

Comment:
Probably better name is `applySlingItemSkill`, because the item may not necessarily be a `Pot`.